voxgraph/include/voxgraph/backend/node/reference_frame_node.h

```diff
@@ -18,9 +18,7 @@ class ReferenceFrameNode : public Node {
   ReferenceFrameNode(const NodeId& node_id, const Config& config)
       : Node(node_id, config), config_(config) {}
 
-  const FrameId getReferenceFrameId() const {
-    return config_.reference_frame_id;
-  }
+  FrameId getReferenceFrameId() const { return config_.reference_frame_id; }
```

Comment:
Hmm I see the recommendation on returning const copies has changed to "don't do it"

Thanks for fixing this.